app/src/main/kotlin/viewmodel/graph/VertexViewModel.kt

```diff
@@ -24,13 +24,9 @@ class VertexViewModel<D>(
         get() = vertex.data.toString()
 
     fun onDrag(dragAmount: DpOffset, currentWindowVM: WindowViewModel, density: Float, scale: Float) {
```

Comment:
In this case, we don't need the density and scale. I recommend trying to fix the issue of different resolutions. In that case, these input variables might be needed.

TOFIX

Reply:
There's no issues with different resolutions, but I agree that there's no need in these parameters and they could be thrown away.

Reply:
GJ